content/en/platform/corda/5.3/deploying-operating/deployment/deploying/_index.md

```diff
@@ -1051,6 +1051,19 @@ For example, when running with Red Hat OpenShift Container Platform, you must us
         name: "corda-privileged"
    ```
 
+4. Override client image for bootstrap vault populating job with specific values recommended by Hashicorp. You must provide this configuration in your `values.yaml` file. Here is an example:
```

Comment:
This isn't related to the Service Accounts; I suggest that this is moved to under the "External Secrets Service" section, possibly under a dedicated sub-header, describing that this is only related to the the Openshift environment

Reply:
Updated